shared/crypto/operations/encrypt/index.tsx

```diff
@@ -63,9 +60,11 @@ const EncryptOptions = (props: EncryptOptionsProps) => {
       />
     </Kb.Box2>
   )
-}
+})
 
 const Encrypt = (props: Props) => {
+  // TODO children should be connected here, don't plumb through all these props for no reason
+  // TODO don't hoist inputValue here. Move it down into TextInput else this whole component renders all the time
```

Comment:
because the state is here, yet not used it forces this entire tree to rerender when the input changes